examples/tv-app/android/java/TVApp-JNI.cpp

```diff
@@ -175,3 +175,27 @@ JNI_METHOD(jboolean, setCurrentLevel)(JNIEnv *, jobject, jint endpoint, jboolean
 {
     return LevelManager::SetLevel(endpoint, value);
 }
+
+JNI_METHOD(jint, addContentApp)
+(JNIEnv *, jobject, jstring vendorName, jint vendorId, jstring appName, jint productId, jstring appVersion, jobject manager)
+{
+    JNIEnv * env = JniReferences::GetInstance().GetEnvForCurrentThread();
+
+    const char * nVendorName = env->GetStringUTFChars(vendorName, 0);
+    const char * nAppName    = env->GetStringUTFChars(appName, 0);
+    const char * nAppVersion = env->GetStringUTFChars(appVersion, 0);
```

Comment:
maybe you can use JniUtfString replace this

Reply:
@nicelyjust Thanks. I have made the change. Can you please review again and approve if it looks ok.